package.json

```diff
@@ -50,9 +50,12 @@
     "lint": "./node_modules/.bin/eslint --ext .js,.jsx --ignore-pattern public .",
     "test": "echo \"Error: no test specified\" && exit 1",
     "format": "prettier --trailing-comma es5 --no-semi --single-quote --write 'src/**/*.js' 'src/**/*.md'",
-    "develop": "gatsby develop",
+    "develop": "npm run dev",
```

Comment:
Do we even need this script if all it does it just run another script?

Reply:
🤷‍♂ You tell me, I just de-duplicated it - maybe it can be removed but someone may use `dev` and another `develop` - no harm in it really

Reply:
Fair.